docs/docs/references/configuration/cli/trivy_image.md

```diff
@@ -39,7 +39,7 @@ trivy image [flags] IMAGE_NAME
       --check-namespaces strings          Rego namespaces
       --checks-bundle-repository string   OCI registry URL to retrieve checks bundle from (default "ghcr.io/aquasecurity/trivy-checks:0")
       --clear-cache                       clear image caches without scanning
-      --compliance string                 compliance report to generate (docker-cis)
+      --compliance string                 compliance report to generate (docker-cis-1.6.0)
```

Comment:
Is it breaking because of this change?

Reply:
I know thats why I put `feat(k8s)!`

Reply:
Yes, I was wondering how it is breaking. Are you going to announce?

Reply:
yes

Reply:
I think all naming updates should be announced as it will break user commands.